ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -857,11 +847,14 @@ evalFunctionCall
    : evalFunctionName LT_PRTHS functionArgs RT_PRTHS
    ;
 
-// cast function
+
+// cast, tostring function
 dataTypeFunctionCall
    : CAST LT_PRTHS logicalExpression AS convertedDataType RT_PRTHS
+   | TOSTRING LT_PRTHS functionArgs RT_PRTHS
```

Comment:
it could be eval function, then use generic eval function implementation approach.